src/java/com/phenix/pct/ABLUnit.java

```diff
@@ -92,8 +92,10 @@ public void addConfiguredFileset(FileSet set) {
         testFilesets.add(set);
     }
 
-    private void writeJsonConfigFile() throws IOException {
+    private void writeJsonConfigFile() throws IOException, RuntimeException {
         JsonWriter writer = null;
+        String foobar = "abc";
+        String foobar2 = "def";
         try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this "try" to a try-with-resources. (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS2093)